gmail/src/utils/http.ts

```diff
@@ -0,0 +1,88 @@
+import { State } from "../models/state";
+
+/**
+ * Make a JSON RPC call with the following parameters.
+ */
+export function postJsonRpc(url: string, data = {}, headers = {}) {
+    // Make a valid "Odoo RPC" call
+    data = {
+        id: 0,
+        jsonrpc: "2.0",
+        method: "call",
+        params: data,
+    };
+
+    const options = {
+        method: "post",
+        contentType: "application/json",
+        payload: JSON.stringify(data),
+        headers: headers,
+    };
+
+    try {
+        const response = UrlFetchApp.fetch(url, options);
+
+        const responseCode = response.getResponseCode();
+
+        if (responseCode > 299 || responseCode < 200) {
```

Comment:
Mmh ? Isn't it a bit yolo-level management of errors ?

Reply:
There's no other way to know if the request succeed or not :/ (there's no response.ok like in python)
See: https://developers.google.com/apps-script/reference/url-fetch/http-response